packs/feat-effects/effect-divine-invulnerability.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_id": "OWFMglWgeCjRhqLK",
+    "img": "icons/magic/holy/barrier-shield-winged-blue.webp",
+    "name": "Effect: Divine Invulnerability",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Divine Invulnerability]</p>\n<p>You gain resistance to all damage equal to half your level.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 14
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder War of Immortals"
+        },
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Godling.DivineInvulnerability.IncreasedResistanceToggle",
+                "option": "divine-invulnerability-increased-resistance",
+                "toggleable": true
+            },
```

Comment:
I hate having two effects, but this second part is a timed thing so we should do an effect for it. Maybe we can ItemAlteration the link into the description of domain spells? Though, maybe impossible to do that properly currently so a potential future thing.

Reply:
Effect added. The note would be a good idea, I would need to think about it some more, but at first glance it doesn't seem like we can narrow down the predicate for it quite enough